ouroboros-consensus/src/Ouroboros/Consensus/HardFork/Combinator/Embed/Binary.hs

```diff
@@ -124,6 +123,17 @@ protocolInfoBinary protocolInfo1 eraParams1 toPartialConsensusConfig1 toPartialL
     shape :: History.Shape '[blk1, blk2]
     shape = History.Shape $ exactlyTwo eraParams1 eraParams2
 
+
+blockForgingBinary :: forall blk1 blk2 m.
+                      ( Monad m
+                      , CanHardFork '[blk1, blk2]
+                      )
+                   => m [BlockForging m blk1]
+                   -> m [BlockForging m blk2]
+                   -> m [BlockForging m (HardForkBlock '[blk1, blk2])]
+blockForgingBinary blockForging1 blockForging2 =
+          alignWith alignBlockForging <$> blockForging1 <*> blockForging2
```

Comment:
Minor: style is just a 4 space indent

Reply:
I think you use the same style as we do, 6 when there is a where clause indented with 4 spaces.

Reply:
We use 4 above and below `where`. Thanks!

![image](https://user-images.githubusercontent.com/1106131/172894384-e4b8d112-f9e5-4800-a939-6cde9a938d3c.png)